setup/create-wp-project/src/commands/theme.js

```diff
@@ -36,6 +36,39 @@ exports.handler = async (argv) => {
 	await clearConsole();
 	await writeIntro();
 
+	const fs = require('fs');
+
+	const findProjectRoot = (currentDir) => {
```

Comment:
how will this work with the plugin flag?

Reply:
I think Plugins to be in a separate task, but putting everything together into one PR can also work, especially when the changes are closely connected and won't get confusing. So, we can go ahead with that approach :).